src/02_spec/07_modules/dem_uart/systemif.rst

```diff
@@ -0,0 +1,53 @@
+System Interface
+----------------
+
+The following signals are required from a bus in order to connect to the DEM.
+
++-----------------+----------+---------------+-------------------------------------------------+
+| Signal          | Width    | Direction     | Description                                     |
++=================+==========+===============+=================================================+
+| ``bus_req``     |    1     | CPU->DEM      | ``1`` indicates that slave is currently selected|
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_addr``    |    3     | CPU->DEM      | Address to be used with write/read operation    |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_write``   |    1     | CPU->DEM      | ``1`` = read, ``0`` = write                     |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_wdata``   |    8     | CPU->DEM      | Read data                                       |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_ack``     |    1     | DEM->CPU      | Acknowledge last read/write operation           |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_rdata``   |    8     | DEM->CPU      | Write data                                      |
++-----------------+----------+---------------+-------------------------------------------------+
+
+The following signals are required from the wishbone bus in order to connect to the DEM using
```

Comment:
Please make this a subsection and explain that this WB interface wraps around the generic bus interface described above.